### PR TITLE
Release Brahma v0.0.2

### PR DESCRIPTION
**Pull request by hariharan-devarajan**

### CODE REVIEW

File: var/spack/repos/builtin/packages/brahma/package.py [hunk 1]
```diff
@@ -16,10 +16,13 @@ class Brahma(CMakePackage):
     version("develop", branch="dev")
     version("master", branch="master")
     version("0.0.1", tag="v0.0.1", commit="15156036f14e36511dfc3f3751dc953540526a2b")
+    version("0.0.2", tag="v0.0.2", commit="1bac58d5aa8962a5c902d401fbf8021aff9104d3")
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/brahma/package.py]: Is this the correct commit?  It looks like the commit starts: `bac58d5`, not with a `1`.

Comment (tldahlgren) [inline var/spack/repos/builtin/packages/brahma/package.py]: By convention, newer versions are listed first.  See note at https://spack.readthedocs.io/en/latest/packaging_guide.html#versions-and-fetching.

**Review by tldahlgren — CHANGES_REQUESTED** [review]
Please see notes.

File: var/spack/repos/builtin/packages/brahma/package.py (hunk 1 above)
Comment (hariharan-devarajan) [reply var/spack/repos/builtin/packages/brahma/package.py]: Fixed.

Comment (hariharan-devarajan) [reply var/spack/repos/builtin/packages/brahma/package.py]: fixed

**Review by tldahlgren — APPROVED** [review]
Confirmed the version commit, rest LGTM.

**Pull request merged**